src/app/components/Timestamp/README.md

```diff
@@ -1,27 +1,33 @@
 <!-- Please remove all references to Simorgh from this when moving the Timestamp component to Psammead! -->
-# (package name) &middot; (badges)
+
+# `(package name) &middot; (badges)`
 
 ## Description
+
 Displays a timestamp, with suitable semantic markup.
 
 ## When to use this component
+
 For displaying an isolated timestamp, separate from the other content in an article or cell.
 
 ## When not to use this component
+
 When a date or time is to be displayed inline inside a paragraph.
 
 ## Installation
+
 `npm install (package-name)`
 
 ## Props
 
-| Argument   | Type   | Required | Default | Example                  |
-|------------|--------|----------|---------|--------------------------|
+| Argument   | Type                                                                                                               | Required | Default | Example                  |
```

Comment:
Prettier does this. It's annoying! Could you change it back to how it was - can add a `<!-- prettier-ignore -->` immediately above the table and Prettier will ignore it.